no-computer.md

```diff
@@ -58,12 +92,21 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Very little time would be spent on the language aspects - it's already done! I
+suppose if someone has a revolutionary idea for transcribing chemical reactions
+they could do so, but I don't know that there is a much better system.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Fairly ambitious - there are lots of kinds of chemistry that use lots of kinds
+of notation.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+It could be a good idea because you'd have to think really, really hard about 
+language design in order to improve it.  It might be a bad idea because that 
+might end up being too hard.
```

Comment:
Again, figuring out what the deliverable would be is a challenge. Especially since this language isn't a programming language, per se, I don't know what the output of a 'program' written in it would look like.